### PR TITLE
No reason for having a 64bit big endian rather an int for the feedback

### PR DESCRIPTION
**Pull request by reno-**
This was the case for the Aux Selection Input Feedback, as referenced in issue https://github.com/SteffeyDev/atemOSC/issues/245

### CODE REVIEW

**Comment by reno-** [comment]
Include also OSC receivers for mask, and feedback too

**Comment by reno-** [comment]
@SteffeyDev , I'm sorry it is my first Pull Request ever on a repo, I did not create a dedicated branch for each feature or fix.

**Comment by SteffeyDev** [comment]
Thanks for the PR! A few things we'll need to do before I can merge this:
1. Please rebase your changes on my develop branch, we need to merge into the develop branch so that the master branch continues to reflect exactly what is deployed in the latest version
2. Please update the README file to reflect the feedback commands you are adding (you'll see at the bottom of the Upstream Keyers section where I list out what feedback is sent)
3. Please add your function to the `UpstreamKeyerMonitor::sendStatus` function, for consistency

File: atemOSC/FeedbackMonitors.mm [hunk 1]
```diff
@@ -366,6 +421,21 @@
 		case bmdSwitcherKeyEventTypeTypeChanged:
 			updateUSKType();
 			break;
+		case bmdSwitcherKeyEventTypeMaskedChanged:
+			updateUSKMasked();
+			break;
+		case bmdSwitcherKeyEventTypeMaskTopChanged:
+            updateUSKMaskTop();
```
Comment (SteffeyDev) [inline atemOSC/FeedbackMonitors.mm:428]: Looks like these spaces need to be changed to tabs

**Pull request closed without merging**